vj.java

```diff
@@ -0,0 +1,61 @@
+package com.muni.myinputcontrols;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.RadioButton;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+public class MainActivity extends AppCompatActivity {
+EditText et_nameobj,et_mobileobj,et_emailobj;
+RadioButton m,f;
+Spinner spin;
+CheckBox j,a,p;
+String gender;
+String lanuage;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
By seeing this file. I became homeless. My Wife along with my second one kicked me out. Thanks for this file. Now i will commit suicide.

Reply:
Sir do you want any requirements?

Reply:
Search for a good wife for me.